q2_quality_control/plugin_setup.py

```diff
@@ -46,9 +46,13 @@
         'method': ('Alignment method to use for matching feature sequences '
                    'against reference sequences'),
         'perc_identity': ('Reject match if percent identity to reference is '
-                          'lower. Must be in range [0.0, 1.0]'),
-        'evalue': 'BLAST expectation value (E) threshold for saving hits.',
-        'threads': 'Number of jobs to execute if method == vsearch',
+                          'lower. (or if E value is higher than E value '
+                          'threshold.)  Must be in range [0.0, 1.0]'),
+        'evalue': ('BLAST expectation (E) value threshold for saving hits. '
+                   'Reject if E value is higher than threshold (or '
+                   'perc_identity is lower than identity threshold.)'),
```

Comment:
The note about `perc_identity` seems misplaced here. I would instead note in the `description` that a search result must meet all of the specified criteria to be considered a match.

Reply:
👍